### ISSUE TITLE
Azure devops pipeines - Globbing not working

### ISSUE THREAD

**Issue by greengumby** [vcsjones/AzureSignTool#283]
Hi I am calling azuresigntool through command line on a hosted agent  version : 6.0.0
azuresigntool sign -kvu **** -kvc ***** -kvi **** -kvs *****  --azure-key-vault-tenant-id **** -tr http://timestamp.globalsign.com/tsa/advanced -td sha256 -v "D:\a\1\a\*.exe"

and the result is "At least one file must be specified to sign."

I can confirm that the file exists and if I fully qualify the file path and name than the signing works as intended.  However I cannot do this when the exe name is dynamic. 

I am also tried "D:\**\*.exe"  with the same result. 

Cheers

**Comment by elksson** [comment]
I am also observing this issue

**Comment by bstordrup** [comment]
I guess this is not related to pipelines - the question is whether AzureSignTool should support signing files based on a file mask - i.e. sign all files matching `MyPath\*.dll`

Personally, I see that as a great feature if you include other signable files that should be signed or if a project outputs more than one binary file (for example .net 9 console apps produce both a .dll and a .exe)

**Comment by vcsjones** [comment]
It should support globbing like that. Let me try to reproduce it.

**Comment by carkar** [comment]
Seeing the same here on 6.0.1

**Comment by manu-st** [comment]
Yes, the issue occurs as soon as you use a drive letter for the file you want to sign. I tried:
c:\...
\\?C:\...
\\localhost\C$\...

In all the above cases it fails. If I use a relative path that goes back to the root of the drive it works fine.
